sway/handlers.c

```diff
@@ -30,6 +32,37 @@
 
 /* Handles */
 
+static bool handle_input_created(struct libinput_device *device) {
+	const char *identifier = libinput_dev_unique_id(device);
+	sway_log(L_DEBUG, "handle_input_created(%s) = %s", identifier, libinput_device_get_name(device));
```

Comment:
sway_log(L_INFO, "Found input device %s", libinput_device_get_name(device));

This should be a higher log priority and should be user friendly, so you can look through your sway log for the input names.